### DISCUSSION TITLE
Is there a shortcode for cutting blog preview text?

### DISCUSSION THREAD

**Discussion by amochkin** [imfing/hextra#148] [General]
I am looking for a way to cut preview text that is displayed on blog post list. By default it seems to cut first 300 characters.

**Comment by amochkin** [comment]
Looking into the code I found it:

```
<!--more-->
```

**Reply by imfing** [reply]
@amochkin also check out hugo documentation:

https://gohugo.io/content-management/summaries/

there are some alternative options  as well
